modules/odc-creating-apiserversource.adoc

```diff
@@ -9,16 +9,12 @@
 
 . Navigate to the **Add** page and select **Event Source**.
 . In the **Event Sources** page, select **ApiServerSource** in the **Type** section.
-+
-image::apiserversource-odc.png[Create an ApiServerSource]
 . Configure the **ApiServerSource** settings:
 .. Enter `v1` as the **APIVERSION**, and `Event` as the **KIND**.
 // .. Select **Resource** as the **Mode**. **Mode** is the mode that the receive adapter controller runs in. `Ref` sends only the reference to the resource. `Resource` sends the full resource.
 // TODO: clarify what this is used for. Out of scope for this PR since not required.
 .. Select the **Service Account Name** for the service account that you created.
-+
-image::apiserver-odc-config-1.png[Select the Service Account Name and Sink]
-.. Select the targeted Knative service from the dropdown menu in **Sink** -> **Knative Service**.
+.. Select the **Sink** for the event source. A **Sink** can be either a **Resource** (Broker, Channel or Knative Service) or a **URI**.
 // .. Add details about the image:
 // General -> Application Name, Name
 // image::apiserver-odc-config-2.png[Configuring the ApiServerSource]
```

Comment:
If we're deleting the image can you also delete these notes I had about it? :slightly_smiling_face: thanks